using-mobile-extensions/adobe-analytics/README.md

```diff
@@ -694,7 +800,19 @@ MobileCore.updateConfiguration(data);
 ```
 {% endtab %}
 
-{% tab title="iOS" %}
+{% tab title="iOS (AEP 3.x) %}
+#### Swift
+
+**Example**
+
+```swift
+let updatedConfig = ["analytics.server":"sample.analytics.tracking.server",
+                     "analytics.rsids":"rsid1,rsid2",
+                     "analytics.batchLimit":10,
+                     "analytics.offlineEnabled":true] as [String : Any] as [String : Any]
```

Comment:
This might be a larger discussion, but how do we feel about updating the spacing in the data type to:
`[String: Any]`
as opposed to having the space on both sides of the `:`?
Also, is it necessary to have two `as [String: Any]` type casts? Should this be updated to a single cast?